mlflow/server/handlers.py

```diff
@@ -335,8 +335,7 @@ def _list_experiments():
 
 @catch_mlflow_exception
 def _get_artifact_repo(run):
-    store = _get_store()
-    return get_artifact_repository(run.info.artifact_uri, store)
+    return get_artifact_repository(run.info.artifact_uri)
```

Comment:
BTW, I think (as a result of us having separate `_get_store` methods in handlers.py for the OSS server and in tracking/utils.py) this interestingly results in a new feature where an OSS tracking server could potentially list & allow downloading artifacts from DBFS if it (the server process) has credentials set up correctly. It is kinda cool in that it makes DBFS usable with the OSS server just as any other artifact repository (S3, GCS, etc). There's no action required from this comment, just wanted to point this out / write out my thought process to confirm there aren't any other implications of this change

Reply:
Actually, nvm...I guess artifact repository initialization would still fail on the server after this code change because [_get_host_creds_from_default_store](https://github.com/mlflow/mlflow/pull/1138/files#diff-2672ddebdaf09c4e3cab6ece4ca465c4R137) would fail on the server's store not being a RestStore, so there's no behavior change :)